game_of_life/rendering.py

```diff
@@ -0,0 +1,30 @@
+from pygame import Surface
+import pygame
+
+from game_of_life import const
+
+
+class Renderer():
+    """Represents a renderer used to draw cells on a window"""
+    screen: Surface
+
+    def __init__(self, screen: Surface) -> None:
+        self.screen = screen
+
+    def draw_game(self, state: list[list[int]]) -> None:
+        """Draws the game screen given the current state"""
+        self.screen.fill(const.BLACK)
+
+        for row_index, cell_row in enumerate(state):
+            for column_index, cell in enumerate(cell_row):
```

Comment:
Dommage d'itérer de nouveau sur le tableau, alors qu'on l'a déjà fait pour le calcul. Peut-on trouver une astuce pour éviter ça ?